src/components/DandisetsPage.vue

```diff
@@ -115,6 +141,8 @@ export default defineComponent({
     // https://next.router.vuejs.org/api/#useroute
     const route = ctx.root.$route;
 
+    const showDrafts = ref(false);
+    const showEmpty = ref(false);
     const sortOption = ref(Number(route.query.sortOption) || 0);
     const sortDir = ref(Number(route.query.sortDir || -1));
     const page = ref(Number(route.query.page) || 1);
```

Comment:
I noticed a bug, to reproduce in the deploy preview:
- go to public dandisets
- turn on "Drafts" and "Empty Dandisets"
- Go to a page higher than 2
- turn off "Drafts" and "Empty Dandisets"

I think the fix is to just reset the `page` variable to 1 whenever `showDrafts` or `showEmpty` change.